apps/web/src/components/layout/components/PageContainer.tsx

```diff
@@ -18,11 +19,18 @@ function PageContainer({
   };
 
   return (
-    <Container pl={0} pr={0} fluid style={containerStyle} h={`calc(100vh - ${HEADER_HEIGHT}px)`}>
+    <StyledContainer pl={0} pr={0} fluid style={containerStyle} h={`calc(100vh - ${HEADER_HEIGHT}px)`}>
       <PageMeta title={title} />
       {children}
-    </Container>
+    </StyledContainer>
   );
 }
 
 export default PageContainer;
+
+const StyledContainer = styled(Container)`
+  overflow-y: auto;
+  border-radius: 0;
+  padding-left: 0;
+  padding-right: 0;
+`;
```

Comment:
style props didnt accepted the overflowY property, son had to create a styled component.

Without overflow-y: auto, the container was getting cut off at 100vh - 40px
<img width="1511" alt="image" src="https://github.com/novuhq/novu/assets/29251776/45e25478-4297-4c24-abcc-dd21839cbf2d">
adding overflow fixes the issue